front-end/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,165 @@
+        minWidth: 200,
+        "& .MuiInput-underline:before": {
+            borderColor: '#009688'
+        },
+        padding: 3
+    },
+    inputLabel: {
+        color: "rgba(0, 0, 0, 0.5393)",
+        "&.Mui-focused": {
+            color: "#599C6D"
+        }
+    },
+    searchIcon: {
+        width: 35,
+        height: 35,
+        backgroundColor: '#009688',
+        padding: 7,
+        borderRadius: '50%',
+        color: '#fff',
+        marginLeft: 58
+    },
+    buttonWrap: {
+        marginLeft: 250
+    }
+}))
+
+const LogoTitle = withStyles({
+    root: {
+        color: '#1261A3',
+        fontWeight: 500,
+        fontSize: '30px',
+    }
+})(Typography);
+
+const NavbarTitle = withStyles({
+    root: {
+        color: '#1261A3',
+        fontWeight: '400',
+        fontSize: '18px',
+        lineHeight: '12px'
+    }
+})(Typography);
+
+const CreateButton = withStyles({
+    root: {
+        backgroundColor: '#1261A3',
+        borderRadius: 3,
+        border: 0,
+        color: 'white',
+        height: 36,
+        padding: '0 20px',
+        marginLeft: '50px',
+        '&:hover': {
+            backgroundColor: '#1261A3'
+        }
+    },
+    label: {
+        textTransform: 'capitalize',
+        fontSize: '14px',
+        fontWeight: '500'
+    },
+})(Button);
+
+export default function Navbar() {
+    const classes = useStyles();
+    const [state, setState] = useState({
+        type: '',
+    });
+    const handleChange = (event) => {
+        const name = event.target.name;
+        setState({
+            ...state,
+            [name]: event.target.value,
+        });
+    };
+
+    return (
+        <div className={classes.root}>
+            <Grid item container className={classes.navbar} alignItems="center">
+                <Grid item container sm alignItems="center" className={classes.logoWrap}>
+                    <Grid item className={classes.logo}>
+                        <img src={logo} className={classes.logoImage} />
+                    </Grid>
+                    <LogoTitle>Civic</LogoTitle>
+                </Grid>
+                <Grid item container sm alignItems="center" className={classes.searchWrap}>
+                    <Grid item>
+                        <FormControl className={classes.formControl}>
+                            <InputLabel htmlFor="type-select" className={classes.inputLabel}>Status</InputLabel>
+                            <Select
+                                native
+                                value={state.type}
+                                onChange={handleChange}
+                                inputProps={{
+                                    name: 'type',
+                                    id: 'type-select',
+                                }}
+                            >
+                                <option aria-label="" value="" />
+                                <option value="Proposed">Proposed</option>
```

Comment:
When the user presses to see the drop down, it is not very elegant, a grey box appears where the value will go. I think this should be the same grey as the background, or perhaps have rounded corners to make it more elegant. Please try a few things to see if this can be a bit more stylish.

Also, the drop-down is not vertically aligned with the search icon. Both the drop-down and the search icon should be vertically aligned centred within the search element (with the white border)